tools/topology/topology2/cavs-sdw.conf

```diff
@@ -69,6 +69,11 @@ Define {
 	DEEP_BUFFER_PIPELINE_SRC        'mixin.15.1'
 	DEEP_BUFFER_PIPELINE_SINK       'mixout.1.1'
 	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	DEEP_BUFFER_PIPELINE_ID_2	16
+	DEEP_BUFFER_PCM_ID_2		35
```

Comment:
the other question is: why add this for LNL only? if this is really useful why not add this capability in all Chrome topologies?

Reply:
As the current topology for sdw, had Deepbuffer option only for "device 31: Deepbuffer Jack Out"
thought of  adding to Spk-amps too..
Dint intend to add for all Chrome devices, as only from MTL, we started to validate Multiple Stream.

Reply:
Is this a real need though? I mean, every deep-buffer stream will eat-up memory that will be reserved, so every time we add such a stream it means less memory for 3rd party libraries. Multiple streams for all endpoints has a cost in the end...

Reply:
Thanks for the explanation about memory impact with this new stream.
In this scenario, we should remove existing "device 31: Deepbuffer Jack Out", until requested by customer ?